src/gui/advancedsettings.cpp

```diff
@@ -376,7 +379,12 @@ void AdvancedSettings::loadAdvancedSettings()
     m_spinBoxAsyncIOThreads.setValue(session->asyncIOThreads());
     addRow(ASYNC_IO_THREADS, (tr("Asynchronous I/O threads") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#aio_threads", "(?)"))
             , &m_spinBoxAsyncIOThreads);
-
+    // stop tracker timeout
+    m_spinBoxStopTrackerTimeout.setMinimum(1);
+    m_spinBoxStopTrackerTimeout.setMaximum(30);
+    m_spinBoxStopTrackerTimeout.setValue(session->stopTrackerTimeout());
+    m_spinBoxStopTrackerTimeout.setSuffix(tr(" s", " seconds"));
+    addRow(STOP_TRACKER_TIMEOUT, tr("Stop Tracker Timeout"), &m_spinBoxStopTrackerTimeout);
```

Comment:
You should add a `?` libtorrrent documentation help link like in the other settings